### PR TITLE
Assist UI improvements

### PR DESCRIPTION
**Pull request by ryanclark**
- Includes billing banners in the page height calculation to remove the scrollbar when the team trial upgrade banner is shown
- Adds the title of the active conversation to the top bar
- Changes the assist landing page
- Rewords the assist tooltip
- Shows an error message if the session history cannot be loaded

image

image

### CODE REVIEW

**Comment by jakule** [comment]
@ryanclark Linter is failing

File: web/packages/teleport/src/Assist/contexts/messages.tsx [hunk 1]
```diff
@@ -222,12 +222,18 @@ async function convertServerMessage(
...
+      Accept: 'text/plain',
+      'Content-Type': 'text/plain; charset=utf-8',
+    });
+
+    let msg;
+    let errorMsg;
+    if (resp.status === 200) {
+      msg = await resp.text();
+    } else {
+      errorMsg = 'Something went wrong';
```
Comment (jakule) [inline web/packages/teleport/src/Assist/contexts/messages.tsx:235]: It probably should be "No session recording. The command execution failed" or something like that. Something went wrong suggest that something (fetching the log for example) failed. I also think that no session recording should return 4xx something.

File: web/packages/teleport/src/Assist/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        <p>
+          Teleport Assist utilizes facts about your infrastructure to help
+          answer questions, generate command line scripts and help you perform
+          routine tasks on target resources.
+        </p>
+
+        <Warning>
+          This is an experimental project. The LLM can hallucinate and produce
+          harmful commands. Do not use in production. Let us know what you think
+          in our community Slack at <strong>goteleport.slack.com</strong>.
```
Comment (benarent) [inline web/packages/teleport/src/Assist/LandingPage.tsx:141]:
```suggestion
          in our community Slack at <strong>https://goteleport.com/slack</strong>.
```

**Review by jakule — APPROVED** [review]
LGTM, one comment

File: web/packages/teleport/src/Assist/contexts/messages.tsx (hunk 1 above)
Comment (ryanclark) [reply web/packages/teleport/src/Assist/contexts/messages.tsx]: It returns a 403 forbidden for some reason I've updated the text

File: web/packages/teleport/src/Navigation/NavigationSwitcher.tsx [hunk 3]
```diff
@@ -295,8 +295,8 @@ export function NavigationSwitcher(props: NavigationSwitcherProps) {
             <TooltipTitle>
               <TooltipTitleBackground>New!</TooltipTitleBackground>
             </TooltipTitle>{' '}
-            Connect Teleport to OpenAI GPT-4 and try out our new Assist
-            integration
+            Try out Teleport Assist, a GPT-4 powered assistant that leverages
```
Comment (reedloden) [inline web/packages/teleport/src/Navigation/NavigationSwitcher.tsx:298]:
```suggestion
            Try out Teleport Assist, a GPT-4-powered AI assistant that leverages
```

File: web/packages/teleport/src/Assist/LandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      <Content>
+        <Title>Teleport Assist</Title>
+
+        <p>
+          Teleport Assist utilizes facts about your infrastructure to help
+          answer questions, generate command line scripts and help you perform
+          routine tasks on target resources.
+        </p>
+
+        <Warning>
```
Comment (reedloden) [inline web/packages/teleport/src/Assist/LandingPage.tsx:138]: Legal has specific language they want to see, so should work with them on final language here.

File: web/packages/teleport/src/Assist/LandingPage.tsx [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+          This is an experimental project. The LLM can hallucinate and produce
```
Comment (reedloden) [inline web/packages/teleport/src/Assist/LandingPage.tsx:139]: Let's not use LLM here. LLM is not a term everybody is going to understand. Maybe "AI" or "AI interface"?

Comment (ryanclark) [reply web/packages/teleport/src/Assist/LandingPage.tsx:141]: This text is from @klizhentas

Comment (klizhentas) [reply web/packages/teleport/src/Assist/LandingPage.tsx:141]: the AI is ok

**Review by klizhentas — APPROVED** [review]

File: web/packages/teleport/src/Assist/LandingPage.tsx (hunk 4 above)
Comment (reedloden) [reply web/packages/teleport/src/Assist/LandingPage.tsx:140]: @klizhentas Legal does have some requirements. I'll follow-up with you separately.

**Pull request merged**